### PR TITLE
feat(wallet-mobile): Remove Yoroi logo from shared image

### PR DESCRIPTION
**Pull request by michalszorad**
Resolves [YOMO-1282](https://emurgo.atlassian.net/browse/YOMO-1282)

[YOMO-1282]: https://emurgo.atlassian.net/browse/YOMO-1282?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by jorbuedo — APPROVED** [review]

**Comment by michalszorad** [comment]
@stackchain I've put this to 4.26.0 feel free to change, if needed

**Comment by michalszorad** [comment]
Waiting for a Product feedback

File: apps/wallet-mobile/src/features/Receive/common/CaptureShareQRCodeCard/CaptureShareQRCodeCard.tsx [hunk 1]
```diff
@@ -5,7 +5,6 @@ import LinearGradient from 'react-native-linear-gradient'
 import QRCode from 'react-native-qrcode-svg'
 
 import {Spacer, Text} from '../../../../components'
-import {YoroiLogoIllustration} from '../../illustrations/YoroiLogo'
 
 type ShareProps = {
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Receive/common/CaptureShareQRCodeCard/CaptureShareQRCodeCard.tsx]: @michaeljscript can't we capture directly on the previous screen? (is the layout the same?)

Comment (michalszorad) [reply apps/wallet-mobile/src/features/Receive/common/CaptureShareQRCodeCard/CaptureShareQRCodeCard.tsx]: Yeah, that's what I would like to do, but waiting for a product feedback

Comment (michalszorad) [reply apps/wallet-mobile/src/features/Receive/common/CaptureShareQRCodeCard/CaptureShareQRCodeCard.tsx]: I have updated it to take screenshot on the same screen, but waiting for product feedback the text flickering is ok

**Comment by michalszorad** [comment]
Product decided not to proceed with these changes

**Comment by stackchain** [comment]
> Product decided not to proceed with these changes

Sad, flickering thing will stick with the code.

**Pull request closed without merging**